main.py

```diff
@@ -30,7 +30,7 @@ def main():
 
     intermediate_score_table = get_intermediate_score_table(result)
     final_score_table = get_final_score_table(intermediate_score_table, user_list)
-    markdown_table = convert_df_to_markdown(final_score_table)
+    # markdown_table = convert_df_to_markdown(final_score_table)
```

Comment:
@Swechhya We don't need this?

Reply:
@mesaugat We currently don't use this. This was added initially because we wanted to show the table in markdown format. We can remove this as of now. I commented it as we may require it in future.

Reply:
@Swechhya Makes sense. Let's remove the code from here. We can add it later if we need it.